geomesa-blobstore/README.md

```diff
@@ -17,3 +17,40 @@ The class `WKTFileHandler` serves as an example of that latter approach.
 
 `FileHandler` implementations are loaded at runtime by the Java ServiceLoader.  As such, available 
 implementations should have their classname registered in `META-INF/services/org.locationtech.geomesa.blob.core.handlers.FileHandler`.
+
+## REST api
+
+The RESTful api for the Blobstore can be easily utilized via cURL as so:
+
+Before we can do anything we must first register a blob store. This typically only needs to be run once in the lifetime
+of the webserver, so if the server is shutdown or restarted the registration call will need to be repeated.
```

Comment:
hadn't gotten to it yet, basically it is only persistent so long as the webserver is running. ie, if i call it and then 5 minutes later kill my tomcat or jetty, then you need to call it again. I think there is a ticket/ pr for something doing something else, unless I was reading too much into it in which case it depends on how you define 'persistent'